install/kubernetes/cilium/charts/agent/values.yaml

```diff
@@ -26,3 +26,9 @@ initResources:
   requests:
     cpu: "100m"
     memory: "100Mi"
+
+## Node tolerations for pod assignment on nodes with taints
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+##
+tolerations:
+- operator: Exists
```

Comment:
nit: one new line and maybe follow the below identation, just for consistency.

```
tolerations:
  - operator: Exists
```

Reply:
Yeah I wasn't sure which nomenclature to chose. I am usually fond of the indented approach but it seems to not be the one use anywhere else 🤔